### ISSUE TITLE
Wrong log_softmax dimension in model

### ISSUE THREAD

**Issue by timmeinhardt** [ikostrikov/pytorch-meta-optimizer#10]
The following line computes the `log_softmax` over the batch dimension (default: `dim=0`):
https://github.com/ikostrikov/pytorch-meta-optimizer/blob/0154d4d4fc856163885a62bac06174311aa58caf/model.py#L19

This should include a `dim=-1` to result in combination with `F.nll_loss` into the cross entropy loss.

**Comment by timmeinhardt** [comment]
Apparently default is `dim=0`. So this issue is obsolete.
